### ISSUE TITLE
"Enhancement Idea" Per Block Selection for doors!

### ISSUE THREAD

**Issue by Superfudge247** [TheTurkeyDev/Call_Of_Minecraft-Zombies#93]
Make door creation similar to barrier creation with a per block system as it would allow for doors to be created that open 2 paths rather than just the one so for example 

-------------/^door/------------------------/^door/---------
                                        ^player^

Which ever sign they use will open both these path ways as they were both selected each block at a time

**Comment by Superfudge247** [comment]
Keep both worldedit style and single block selection have it as a type of door

**Comment by ikbenignace** [comment]
This issue can be closed right, because this feature already exists now @TheTurkeyDev

**Comment by TheTurkeyDev** [comment]
No, the door creation isn't block by block currently, it's selecting an area with 2 points. This was asking for it to allow any shape doors

**Comment by TheTurkeyDev** [comment]
Not hard to implement, and luckily I don't think you'd have to change the area saves at all since iirc, they save every block individually
